chart/templates/secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ template "appname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: {{ .Values.secrets.type }}
+data:
+  {{- range $k, $v := .Values.secrets.env }}
+  {{ $k }}: {{ $v | quote }}
```

Comment:
would it make sense to the the base64 encoding here, instead of forcing the user to do it in the gitlab config?

Reply:
Yes indeed. 38af5c3c6ea506c4889d793129a4b431f0e34626